recipes/_efa_enable.rb

```diff
@@ -13,13 +13,10 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-efa_tarball = "#{node['cfncluster']['sources_dir']}/aws-efa-installer-latest.tar.gz"
-
-# Get EFA Installer
-remote_file efa_tarball do
-  source node['cfncluster']['efa']['installer_url']
-  mode '0644'
-  retries 3
-  retry_delay 5
-  not_if { ::File.exist?(efa_tarball) }
+bash "enable efa limits" do
+  cwd Chef::Config[:file_cache_path]
+  code <<-EFAENABLE
+    cd aws-efa-installer/install
```

Comment:
file_cache_path is a temp dir, so aws-efa-installer could be not present anymore, between the base_install and the base_config recipes

Reply:
changed to sources dir: `/opt/parallelcluster/sources`